benefits/core/models.py

```diff
@@ -213,10 +209,14 @@ def eligibility_api_public_key_data(self):
         """This Verifier's public key as a string."""
         return self.eligibility_api_public_key.data
 
+    @property
+    def supports_claims_verification(self):
+        return bool(self.claims_scope) and bool(self.claims_claim)
```

Comment:
I think we can put this logic inline in `uses_claims_verification` and remove this method.

Reply:
I agree, removing this method made the model a bit simpler 👍